### DISCUSSION TITLE
G/L, Asset Tracking, Inventory Cost Questions

### DISCUSSION THREAD

**Discussion by GlasgowGrip** [blueseerERP/blueseer#113] [Q&A]
Hello, I have past ERP experience from the shop floor/MES point of view, but now I am learning the accounting and general ledger side of things, and am evaluating Blueseer for my startup (parts and repair services, and also light manufacturing).  I just stood-up a Blueseer instance on MySQL over the weekend and have been exploring.  The Inventory management and WO management functionality (i.e. multi-level BOM's, WIP, Routings, scheduling) are highly attractive.  Also the Service Order model (quote-order-invoice) seems to be a pretty neat fit (still investigating).

But...   I am feeling somewhat doubtful that Blueseer is suitable as an overall G/L solution.  Now, it's also **highly** possible I am just too new to the Accounting side of things!

The documentation PDF and tutorial videos on the site are quite helpful.  But still so many questions...
1. Is there a discussion forum or community anywhere else dedicated to Blueseer that I can go to for help/advice?  
2. For one example, I'm basically stumped as to a good approach for tracking Fixed Assets (i.e. for tracking Equipment, and Furniture/Fixtures).  Do I create an Item master for each piece of equipment I buy, of Class "A" (for Asset) and Type "ASSET", and then create a PO and receive it just like Purchased RM inventory?  For example, purchase an Oscilloscope for $1000.  But need guidance on a business process for acquiring that asset and then tracking depreciation over time, and someday disposing of it (either sell it or scrap it).  Is there a process or good workaround using accounts and G/L transactions?  I guess I'm okay if the answer is calculate/track it outside of Blueseer and then enter some kind of yearly depreciation expense transaction, for example.
3. Also unsure on best practice for how to treat Manufacturing Supplies, as well as Office Supplies for that matter.
4. How to handle shipping charges when ordering/receiving a PO with multiple (many) lines.  Any way to distribute shipping cost across all the PO lines received?  Also, any facility to update costs for each item using FIFO, Avg Cost or LIFO models, as more parts are purchased/received over time?
5. This one I can experiment with, but if I invoice against a Service Order, will parts get relieved from Inventory and expensed to COGS account?
6. In my repair business, I will typically charge for parts and labor - a good fit for the Service order process (quote to order to invoice).  What if I also need to charge for shipping a customer's unit back to them after repairs are completed?  I would like to show the shipping cost on the estimate, is there a good way to do that?

I know that's an awful lot - please advise if there is a better way to find answers.  And thanks so much - you've done an incredible job creating this software!  Cheers - Steve

**Comment by vaughnte** [comment]
BlueSeer double entry GL is actually one of it's better features....however, it does not have a FixedAsset solution.   I have had requests for it, but not enough to warrant the time...(that could change though).    I'll try to answer your questions top down : 
1)  As far as a discussion forum...you're on it.  lol   The website and the github project site is all there is with regards to discussions, requrests.   If you have a specific question...feel free to shoot an email to support@blueseer.com.   It will get answered.
2)  sorry...but no F/A solution at this time...you're alternative that you mentioned is probably your best bet.
3)  You can just Miscellaneous Expense any non-inventory items.   See 'Expense Maintenance' under the Finance main menu.   You can itemize each expense and charge to whatever GL Expense account you desire.   Dont forget to 'Post GL' every so often to make sure your acct balances get updated.
4)   Shipping charges on a PO can be handled as a 'summary' level charge to the PO...but it does not get distributed over the line items...it's a bulk summary charge.   All costing is Standard costing.   Current costs are rolled to standard costs at whatever frequency is necessary....usually once a year.
5)  yes....inventory items in Service Order will get relieved from inventory when the service order is committed....along with the appropriate Income Account debit/credit and COGS Account debit/credit
6)  I would add a line item on the 'quote' indicating 'estimated return shipping charges'...and then adjust accordingly at commit time.

I hope this answers your questions...feel free to reach out to either this discussion or send an email to support with any further questions or concerns as you research the package.

**Reply by GlasgowGrip** [reply]
Hi Terry - thanks for the detailed and timely response, you are very very helpful.

To clarify answer 5:  The Invoicing action is the "commit" step you are referring to that would relieve the Inventory and generate the Income/COGS transactions, yes?

Revisiting answer 6:  the estimated shipping charges line item on the quote would be a "service"-type line item, yes?  And when you say "commit time", are you referring to the conversion from Quote to a firm Service Order, or the Invoicing of the Service Order?  I guess either way, that shipping charge line item would be rolled into the total that's charged to the customer, so that sounds good.  But how does the actual shipping Expense get accounted for?  A separate, manual, Expense transaction?  I feel like I might be missing something there...  or maybe not - does that not happen with regular Sales Orders, either?  If so then I made a faulty assumption - maybe the system does not allow for automagically recording shipping charges on either type of order, and always assumes the customer handles freight from their end?

Sorry for being so high maintenance here.  I do thank you again.  Cheers! - Steve

**Reply by vaughnte** [reply]
Hey Steve,   Yes..for answer 5...the commit step is the invoice 'button'...it will close the order and invoice....and relieve any inventory items that are on the service order.
With regards to shipping cost, it currently does not hit a 'separate' shipping specific GL account.   It gets bundled in to the overall value of the order and hits the income statement and AR accounts.   The Sales Order module does the same thing, but it does have separate reporting for summary charges...so if a summary charge with label 'shipping' is added, you can see those charges per order as a standard report...but not at the GL level.   Note...that is some functionality that will be introduced in the next patch release though...as several people have asked about shipping charges landing separate in the GL....as does 'taxes' do now.  However, this will only be at the Sales Order and Purchase Order modules.....as Service Order module does not have summary charges at this time.

**Reply by GlasgowGrip** [reply]
Hi Terry - thank you for the clarifications.  Glad to hear you are adding the shipping G/L account feature to SO's, I believe that is valuable.  I guess my use case is just a little off from what you had envisioned for a "Service Order"...  I would still vote for adding the summary/shipping charge feature to Service Orders, basically identical to what you're doing with standard SO's.  Thanks again! - Steve
